rtuclient.go

```diff
@@ -268,6 +268,8 @@ func (mb *rtuSerialTransporter) Send(aduRequest []byte) (aduResponse []byte, err
 	mb.lastActivity = time.Now()
 	mb.startCloseTimer()
 
+	time.Sleep(mb.charDuration(3.5))
```

Comment:
Alternativ:

	defer func() { mb.lastActivity = time.Now() }()

	// 3.5 characters delay
	if d := time.Until(mb.lastActivity.Add(time.Duration(35000000 / mb.BaudRate))); d > 0 {
		time.Sleep(d)
	}

Reply:
Ausser dass die Formel zur Zeitberechnung nicht stimmt eine gute Idee 👍🏼

Reply:
In my non-competent view we should make the basics right first (such as formulas) and stick to the standards, when such standards exists.